### PR TITLE
Nc games 5

### PR DESCRIPTION
**Pull request by SZahl**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -74,3 +74,37 @@ describe.only('GET /api/reviews/:review_id', () => {
...
+                expect(review).toMatchObject({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_body: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number)
```
Comment (itsaliabdallah) [inline __tests__/app.test.js:96]: You're not checking that there is a comment_count

**Comment by Harrison365** [comment]
Hi Sarah 🙂  

This is looking good so far 👍. However, we need to check for the comment count on each review.

Please let us know when you want us to take another look. Thanks 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -74,3 +74,38 @@ describe.only('GET /api/reviews/:review_id', () => {
...
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_body: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(String)
```
Comment (NRMSMITH) [inline __tests__/app.test.js]: I think comment_count would be better as a number! Have a look at how to change a string to a number :)

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
change comment_count to a number please :D

**Review by NRMSMITH — APPROVED** [review]
stunning 🕺

**Pull request merged**